lib/diagnostic.rb

```diff
@@ -105,15 +125,22 @@ def eat_meat(food)
 end
 
 ##
-# your response here
```

Comment:
This class definition is broken since it needs a closing `end`.  There needs to be a `roar` method that prints `'ROAR!'`.  Currently, the constructor sets an instance variable, `@roar` to the string `'ROAR!'`.

```ruby
class Lion < Cat
  include Carnivorous

  def roar
    puts 'ROAR!'
  end
end
```